python_scripts/04_basic_parameters_tuning.py

```diff
@@ -278,13 +278,13 @@
 # %% [markdown]
 # ## Combining evaluation and hyper-parameters search
 #
-# Cross-validation was used for searching the best model parameters. We
-# previously evaluate model performance through cross-validation as well. If we
+# Cross-validation was used for searching for the best model parameters. We
+# previously we evaluated model performance through cross-validation as well. If we
```

Comment:
There is a we already on the previous line!

Reply:
I will remove it, no worries!

Reply:
whoops....